### ISSUE TITLE
Problème de cache bizarre avec varnish

### ISSUE THREAD

**Issue by wallon-ines** [Drupal-FR/site-drupalfr#69]
Si je passe par varnish j'ai la css complète de eu_cookie_compliance
![css](https://user-images.githubusercontent.com/11542186/37854409-1a8494e6-2eeb-11e8-8017-1d7de6fbc819.png)

Sans varnish, j'ai 

![css_bare](https://user-images.githubusercontent.com/11542186/37854438-383ab22c-2eeb-11e8-92e2-f5b1272e59bf.png)

même après un cr et une désactivation de redis

**Comment by FlorentTorregrosa** [comment]
Salut Inès,

Est-ce que la page a été mise en cache par Varnish avant ?

Au niveau de la conf du varnish purger, il est réglé pour purger lors du cron et non directement, et le drush cr ne va pas vider le cache Varnish. Il y a des commandes Drush, mais pas pour Drush 9.

Redis ne devrait pas avoir d'impact sur ce comportement.

**Comment by wallon-ines** [comment]
Effectivement avec cron ça passe mieux, 

Personnellement lancer le cron pour vider les caches varnish, c'est pas instinctif

**Comment by FlorentTorregrosa** [comment]
Oui, c'est parce qu'il ne faut pas que le drush cr puisse vider le cache Varnish, sinon sur un serveur de prod ça peut faire tomber le serveur. Donc soit lancer le cron qui va dépiler la queue de varnish purge, soit lancer la purge via commande Drush dédiée (mais pas encore compatible Drush 9).
